bezier/src/cubic_bezier.rs

```diff
@@ -183,6 +183,73 @@ impl CubicBezierSegment {
         find_cubic_bezier_inflection_points(self)
     }
 
+    /// Return local y extremum points or None if this curve is monotone.
+    ///
+    /// This returns the advancements along the curve, not the actual y position.
+    pub fn find_y_extremums(&self) -> UpToTwo<f32> {
+        let mut ret = UpToTwo::new();
+        // See www.faculty.idc.ac.il/arik/quality/appendixa.html for an explanation
+        // The derivative of a cubic bezier curve is a curve representing a second degree polynomial function
+        // f(x) = a * x² + b * x + c such as :
```

Comment:
To avoid confusion, let's say that it is the derivative of the curve projected on the y axis, and use `y` as the variable in the formula: `f(y) = a * y² + b * y + c` (because we also need the equivalent for the x axis).